src/pages/_proxied-dot-io/waypoint/docs/[[...page]].tsx

```diff
@@ -4,16 +4,22 @@ import NestedNode from 'components/_proxied-dot-io/waypoint/nested-node'
 import DocsPage from '@hashicorp/react-docs-page'
 import productData from 'data/waypoint.json'
 import { isVersionedDocsEnabled } from 'lib/env-checks'
+import ImageConfigBase from 'components/image-config'
+import { ImageConfigProps } from 'components/image-config/types'
 // Imports below are used in getStatic functions only
 import { getStaticGenerationFunctions } from 'lib/_proxied-dot-io/get-static-generation-functions'
 
+const ImageConfig = (props: ImageConfigProps) => (
+  <ImageConfigBase hideBorder {...props} />
```

Comment:
It's not clear to me yet, why does this one have `hideBorder` but the Vault one doesn't?

Reply:
Awesome, that sounds good!